### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by kaslin**
Adding more info about roles in our group and updating the "Join Us" section.

### CODE REVIEW

**Review by Douglasbinha — APPROVED** [review]

File: communication/contributor-comms/README.md [hunk 1]
```diff
@@ -28,10 +30,10 @@ To better inform the Kubernetes contributor community and highlight their work t
 
 We are always open to being a home for contributors to Kubernetes whether you code or not. 
 
+To find out what projects we're currently working on and explore what you could do to contribute:
 * Come to one of our [meetings] (Every Friday at 8am PT). See the [calendar] to add it to your own calendar.
-* Join our [mailing-list], and/or reach out to us on the #sig-contribex channel on the Kubernetes slack ([slack.k8s.io](http://slack.k8s.io)).
-
-Let us know you are interested and if you have any questions.
+* Reach out to us on the #sig-contribex channel on the Kubernetes slack ([slack.k8s.io](http://slack.k8s.io)).
```
Comment (AvineshTripathi) [inline communication/contributor-comms/README.md:35]: I think we should update channel from sig-contribex to sig-contribex-comms

Comment (kaslin) [reply communication/contributor-comms/README.md]: Should be fixed with my latest commit - sorry I lost track of this!

**Comment by kaslin** [comment]
@AvineshTripathi @chris-short would you take a look at this and see if it's good to go?

**Review by fsmunoz — APPROVED** [review]

**Pull request merged**